srcpkgs/libspnav/template

```diff
@@ -1,27 +1,34 @@
 # Template file for 'libspnav'
 pkgname=libspnav
-version=0.2.3
+version=1.0
 revision=1
 build_style=gnu-configure
 configure_args="--disable-opt"
 makedepends="libX11-devel"
 short_desc="Open source alternative to 3DConnextion drivers"
 maintainer="yopito <pierre.bourgin@free.fr>"
-license="GPL-3"
+license="BSD-3-Clause"
 homepage="http://spacenav.sourceforge.net/"
 distfiles="${SOURCEFORGE_SITE}/spacenav/${pkgname}-${version}.tar.gz"
-checksum=7ae4d7bb7f6a5dda28b487891e01accc856311440f582299760dace6ee5f1f93
+checksum=8849b7f7826d750f6956cf8f4f53937f2359ab6da97d6c834c71d5f771212e7c
 
 do_build() {
 	make CC="${CC}" AR="${AR}"
 }
 
+post_install() {
+	vlicense LICENSE
+}
 libspnav-devel_package() {
 	short_desc+=" - development files"
 	depends="${makedepends} ${sourcepkg}>=${version}_${revision}"
 	pkg_install() {
 		vmove usr/include
 		vmove "usr/lib/*.a"
 		vmove "usr/lib/*.so"
+post_install() {
```

Comment:
copy paste mistake?

Reply:
oops, thanks for catching it. fixed and git pushed force.